src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -117,8 +111,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
-  private EnterprisePlugin initEnterprisePlugin(Boolean isInteractive)
-          throws MojoFailureException {
+  private EnterprisePlugin initEnterprisePlugin(Boolean isInteractive) throws MojoFailureException {
     return isInteractive ? initInteractiveEnterprisePlugin() : initBatchEnterprisePlugin();
```

Comment:
unrelative to this PR, but why `isInteractive` is a `Boolean` (wrapper) instead of `boolean`?

In case of `null`, there will be a NPE in the ternary